src/expandableCalendar/weekCalendar.js

```diff
@@ -177,15 +182,19 @@ class WeekCalendar extends Component {
   };
 
   renderItem = ({item}) => {
-    const {calendarWidth, style, onDayPress, ...others} = this.props;
+    const {calendarWidth, style, onDayPress, markedDates, ...others} = extractComponentProps(Week, this.props);
```

Comment:
`calendarWidth` is not a Week prop... It is a WeekCalendar prop

Reply:
But I don't think you need it here as `this.containerWidth` already reads it

Reply:
Yea you right.. the previous code was `calendarWidth || this.containerWidth` 
I didn't notice it's a getter
Ill fix it

Reply:
Fixed ✅